### ISSUE TITLE
Extend expectEvent to support multiple events of the same type

### ISSUE THREAD

**Issue by nventuro** [OpenZeppelin/openzeppelin-test-helpers#3]
The current helper attempts to match all values on the first event that it finds with a matching name. In the case of a value mismatch, it should keep searching the event list, as opposed to exiting immediately. See for example `PaymentSplitter`'s constructor, where multiple `PayeeAdded` events are emitted in the same transaction.

**Comment by abcoathup** [comment]
This appears to be resolved.

I used the following contract and test

## MyContract.sol
```solidity
// SPDX-License-Identifier: MIT
pragma solidity ^0.6.0;

contract MyContract {

  event DoneStuff(uint256 value);

  constructor() public {
  }

  function doStuff() public {
    emit DoneStuff(1);
    emit DoneStuff(2);
    emit DoneStuff(3);
  }
}
```

## MyContract.test.js
```js
// test/MyContract.test.js
// Load dependencies
const { expect } = require('chai');

// Import utilities from Test Helpers
const { BN, expectEvent, expectRevert } = require('@openzeppelin/test-helpers');

// Load compiled artifacts
const MyContract = artifacts.require('MyContract');

// Start test block
contract('MyContract', function ([ creator, other ]) {

  beforeEach(async function () {
    this.myContract = await MyContract.new({ from: creator });
  });

  it('it emits event when doing stuff', async function () {
    const receipt = await this.myContract.doStuff();

    expectEvent(receipt, 'DoneStuff', { value: new BN('3')});
    expectEvent(receipt, 'DoneStuff', { value: new BN('2')});
    expectEvent(receipt, 'DoneStuff', { value: new BN('1')});
  });
});
```
Running the test:
```
$ npx truffle test

Compiling your contracts...
===========================
> Compiling ./contracts/Migrations.sol
> Compiling ./contracts/MyContract.sol
> Artifacts written to /tmp/test--8142-pc3Bc5J5OvBZ
> Compiled successfully using:
   - solc: 0.6.12+commit.27d51765.Emscripten.clang



  Contract: MyContract
    ✓ it emits event when doing stuff (103ms)


  1 passing (281ms)
```

**Comment by frangio** [comment]
Fixed in #17.
